pkg/restore/restore.go

```diff
@@ -454,6 +459,70 @@ func (ctx *context) execute() (Result, Result) {
 		}
 	}
 
+	// TODO: Re-order this logic so that CRs can be prioritized in the main loop, rather than after.
+
+	// Refresh and resolve based on CRDs added to the API server from the above restore loop.
```

Comment:
Let's add a more extensive comment here explaining what's going on for the benefit of future us :)